### ISSUE TITLE
Doesn't work when elements are placed outside a page div

### ISSUE THREAD

**Issue by robbbyc** [godesign/nativeDroid2#38]
I created a multipage jquery mobile app (all page divs are in 1 file) and the header div is placed outside the page divs so it can be reused for all pages. The nativedroid styling isn't applied to the elements outside pages.
Can this be fixed or can you advise me how to fix this myself in my project ?

Thx

**Comment by godesign** [comment]
You are right, the styling selector may goes directly into the "pages"-context.

You could try to use the "nd2-include" inside all pages and load a separate .html-File. So you only have to maintain the content at one place.

Using nd2-include is pretty easy (in the demo used for the left sidebar):

``<nd2-include data-src="/examples/fragments/panel.left.html"></nd2-include>``

Documentation:
http://nativedroid.scripter.click/nd2-include/

**Comment by robbbyc** [comment]
The nd2-include tag doesn't work in IIS. The error I recieve is http error 405 (method not allowed). One solution I found via Google works but requires using the asp isapi extension. I'm planning to publish my html 5 app to the Windows store and I don't think I can configure an isapi extension in a web app.

I also tried including the header html via jquery in the pagebeforecreate event. This works but the overlay menu only appears once per page.

Is it possible to change the nativedroid code to also work for headers outside a page div ?

**Comment by janrg** [comment]
I've tried using your solution with nd2-include, but after clicking back and forth a few times, I always end up with two copies of <div class="ui-panel-dismiss"></div> and then the next time the panel is opened, one of them doesn't get closed and hangs as an invisible layer over all content, making it impossible to click on anything.
